src/shared-components/dropdown/test.tsx

```diff
@@ -81,6 +83,8 @@ describe('<DesktopDropdown />', () => {
         onSelectClick={() => null}
         closeDropdown={() => null}
         onOptionClick={() => null}
+        textAlign="left"
```

Comment:
Mobile and Desktop Dropdowns are never meant to be used directly those values are now required because I know we have those available in the index. These are just fillers to make the TS pass and has no impact on test or snapshots. 
We never use Mobile/Desktop dropdowns directly